api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -2602,3 +2602,12 @@ entityFiltersSchemas:
               int64Val: 2000000009
             - displayName: "Unknown"
               int64Val: 0
+entityIndexing:
+  - entity: condition
+    allNodesWorkflow:
+      outputTable: aou_synthetic_SR2019q4r4_indexes.condition_all_nodes
```

Comment:
Per your PR description, definitely need to figure out a way to connect this to the rest of the underlay; a fair amount of duplication and room for error with this approach.